web/src/lib/components/utilities-page/duplicates/duplicate-asset.svelte

```diff
@@ -88,13 +106,53 @@
   </div>
 
   <div
-    class="grid place-items-center gap-y-2 py-2 text-xs transition-colors {isSelected
+    class="grid place-items-start gap-y-2 py-2 text-xs transition-colors {isSelected
       ? 'text-white dark:text-black'
       : 'dark:text-white'}"
   >
-    <span class="break-all text-center">{asset.originalFileName}</span>
-    <span>{getAssetResolution(asset)} - {getFileSize(asset)}</span>
-    <span>
+    <div class="flex items-start gap-x-1">
+      <Icon icon={mdiImageOutline} size="16" />
+      <div>
+        <span class="break-all text-center">{asset.originalFileName}</span><br />
+        {getAssetResolution(asset)} - {getFileSize(asset)}
+      </div>
+    </div>
+    <div class="flex items-start gap-x-1">
+      <Icon icon={mdiCalendar} size="16" />
+      {#if dateTime}
+        {dateTime.toLocaleString(
+          {
+            month: 'short',
+            day: 'numeric',
+            year: 'numeric',
+          },
+          { locale: $locale },
+        )}
+
+        {dateTime.toLocaleString(
+          {
+            // weekday: 'short',
+            hour: 'numeric',
+            minute: '2-digit',
+            timeZoneName: timeZone ? 'shortOffset' : undefined,
+          },
+          { locale: $locale },
+        )}
+      {:else}
+        {$t('unknown')}
+      {/if}
+    </div>
+
+    <div class="flex items-start gap-x-1">
+      <Icon icon={mdiMapMarkerOutline} size="16" />
+      {#if locationParts.length > 0}
+        {locationParts.join(', ')}
+      {:else}
+        {$t('unknown')}
+      {/if}
```

Comment:
This is really just a personal preference, but I'd probably do this in JS land, i.e.,
```ts
{locationParts.length > 0 ? locationParts.join(', ') : $t('unknown')}
```